app/assets/javascripts/components/Columns/ColumnItem.js

```diff
@@ -1,20 +1,51 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from 'react'
+import PropTypes from 'prop-types'
+import { DropTarget } from 'react-dnd'
+import { connect } from 'react-redux'
 
-const Column = ({ title, children, renderAction, onClose }) => (
-  <div className="Column">
-    <div className="Column__header">
-      <h3 className="Column__name">{title}</h3>
-      <div className="Column__actions">
-        {renderAction()}
-        <button type="button" className="Column__btn-close" onClick={onClose}>
-          <i className="mi md-light md-16">close</i>
-        </button>
+const columnTarget = {
+  canDrop({ column }) {
+    if(column === 'done') {
+      return false
+    }
+    return true;
+  },
+
+  drop({ column }, monitor) {
+    const story = monitor.getItem();
+    return {...story, column};
+  },
+}
+
+const collect = (connect, monitor) => {
+  const info = {
+    connectDropTarget: connect.dropTarget(),
+    canDrop: monitor.canDrop(),
+  }
+  return info;
+}
+
+const Column = ({
+  title,
+  children,
+  renderAction,
+  onClose,
+  connectDropTarget,
+}) =>
+  connectDropTarget(
```

Comment:
What do you think about we change this to implement `useDrop` like we do here https://github.com/Codeminer42/cm42-central/blob/1cb78294b7ebcada571abf6ede8c476ea86c513d/app/assets/javascripts/components/story/CollapsedStory/index.js#L49